JASP-Engine/JASP/R/principalcomponentanalysis.R

```diff
@@ -605,54 +662,51 @@ PrincipalComponentAnalysis <- function(dataset = NULL, options, perform = "run",
 
 
 # Factor correlations:
-.getFactorCorrelationsPCA <- function(analysisResults, options,perform){
-  
-  # Extract loadings:
-  if (!is.null(analysisResults)){
-    
-    corMatrix <- as.matrix(analysisResults$r.scores)
-    # loadingsMatrix <- matrix(unlist(loadingsMatrix),nrow(loadingsMatrix),ncol(loadingsMatrix))
-    
-    
-    # loadingsMatrix <- matrix(1,12,1)
-    
-  } else {
-    
+.getFactorCorrelations <- function(analysisResults, options, perform, type){
+
```

Comment:
@hummyhummy in a common file

Reply:
this is a common function in only in file principalcomponentanalysis.R